pkg/providers/vsphere/config/template-cp.yaml

```diff
@@ -88,35 +88,6 @@ spec:
       thumbprint: '{{.thumbprint}}'
 {{- end }}
 ---
```

Comment:
So `Cluster` is considered part of CP? I am wondering if we want to template some of that info out too, or change cp to `Core` or something.

Reply:
I thought about it once but `Core` feels confusing to me a bit. CP might not be the best term but it is less confusing.

Reply:
I think the ideal separation is to have three, cp, worker, and core/cluster. But I think that's a lot more scope imo. I think of core because it includes external etcd as well and is needed to operate a kubernetes cluster, but yea I get the confusion.